src/libs/elektra/keyname.c

```diff
@@ -1631,25 +1698,28 @@ ssize_t keySetBaseName (Key * key, const char * baseName)
 }
 
 /**
- * For currently valid namespaces see #elektraNamespace.
+ * Returns the #elektraNamespace for a Key.
```

Comment:
General note: we should update the docu of `#elektraNamespace`, maybe link to Key Name docu (`doc/KEYNAMES.md`). The Key Name docu explains namespaces with more detail.